app/views/verify/fail.html.slim

```diff
@@ -1,10 +1,12 @@
 - title t('idv.titles.hardfail')
+.clearfix
+  .col-10.sm-col-12.mx-auto
+    .center
```

Comment:
do we need all three wrapping divs?

Reply:
That was copy pasta, no, just two should work